clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/caching/agents/TitusStreamingUpdateAgent.java

```diff
@@ -217,6 +217,13 @@ public void executeAgent(Agent agent) {
                   updateJob(state, notification.getJobUpdate().getJob());
                   break;
                 case TASKUPDATE:
+                  if (notification.getTaskUpdate().getMovedFromAnotherJob()) {
+                      Task task = notification.getTaskUpdate().getTask();
+                      String destinationJobId = task.getJobId();
+                      String sourceJobId = task.getTaskContextOrDefault("task.movedFromJob", null);
+                      log.info("{} task moved from another job ", sourceJobId);
```

Comment:
May be helpful to include the taskId and source/dest job ids, something like "task {} moved from job {} to {}".

Reply:
done